response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,52 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+
+<h1 class="saturn" name="page-messages-title">{{ messages[0].get('subject') }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">RU Ref: </span><span id="ruref">    {{ messages[0].get('ruref') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Business: </span><span id="ru">{{ messages[0].get('business') }}</span></div>
+         <div class="secure-message--recipient venus"><span class="sm-form-recipient-label">To: </span><span id="recipient">    {{ messages[0].get('to') }}  </span></div>
+
+    </div>
+    {% for msg in messages %}
+    {% if msg.get('internal') == True %}
+    <div id="smC1M1" class="secure-message-sent-message">
+        <div class="secure-message-sent-message-meta sm-from-ons">
+    {% else %}
+    <div id="smC1M2" class="secure-message-sent-message">
+        <div class="secure-message-sent-message-meta sm-from-respondent">
+    {% endif %}
+            <span class="secure-message-sent-message-meta-from venus ">
+                <span class="tempHack">From:</span>
```

Comment:
tempHack?

Reply:
tempHack was used on the UX guys template which I used as a template. This is what it does in the css:
.tempHack{
  width: 2.5rem;
  display: inline-block;
}

Reply:
Can we update the css from tempHack to 
```
secure-message-sent-message-meta-from > span {
  width: 2.5rem;
  display: inline-block;
}
```

From https://stackoverflow.com/questions/15853899/styling-span-within-a-div